### ISSUE TITLE
Add `--latest` flag to `tuono new` to download latest templates

### ISSUE THREAD

**Issue by Valerioageno** [tuono-labs/tuono#262]

**Comment by gravitypowerful** [comment]
I will do it.

**Comment by gravitypowerful** [comment]
I implemented it.

PR #292

**Comment by jacobhq** [comment]
I can take this from here :)

**Comment by jacobhq** [comment]
I think we should perhaps rename the flag, since the behavior of `latest` in package managers like npm is to get the latest tag, and ignore local cache. Since we don't cache locally, we do this by default anyway.

Maybe we should rename the flag to `--latest-commit`, `--canary` or similar to convey that the user will not be using the latest release, but the latest commit.

WDYT

**Comment by Valerioageno** [comment]
I think `--latest-commit` is the more semantically correct! @marcalexiei WDYT?

> This feature most likely won't be used by the users since it is more for us to check that the experimental experiments ( :) ) are working.

**Comment by marcalexiei** [comment]
Good question!
`--latest-commit` surely sounds correct,
however is very verbose and I don't recall any cli using a command like this 😅

How about using something that match git dictionary like `--head`?

**Comment by Valerioageno** [comment]
Yeah, we can use `--head` and then adding in the inline description something like: `load the latest version available on the main branch`
